test/netpol-conformance/adminnetworkpolicy_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package netpol_conformance
```

Comment:
It's rare to have "-" in package path and "_" in package name. Since this is essentially e2e tests. Can we put it into the netpol directory of e2e, i.e. test/e2e/adminnetworkpolicy/conformance.go, or test/e2e/netpol/conformance.go

Reply:
Done